### PR TITLE
Docs: Add new Ellipsoids page to explain ellipsoidal parameters.

### LINKED ISSUE #1064: Document ellipsoid parameters
The [source code for `pj_ell_set.c`](https://github.com/OSGeo/proj.4/blob/master/src/pj_ell_set.c#L36-L64) describes two ellipsoid size parameters ("a", "R"), five ellipsoid shape parameters ("rf", "f", "es", "e", "b"), and seven spherification parameters ("R_A", "R_V", "R_a", "R_g", "R_h", "R_lat_a", "R_lat_g").

Currently I see `+a`/`+b` documented in `usage/projections.rst`, and `+R` via `operations/options/R.rst`.

This task is to make a section and table of ellipsoid parameters in the docs (possibly somewhere under `usage/`).

### PR DESCRIPTION
**Pull request by direvus**
This PR is a continuation of #2385, which appears to have been abandoned by its original author.

I haved merged the commits from that PR, incorporated the PR feedback, and made a few changes and improvements of my own.

I hope you find this helpful.

 - [x] Closes #1064 
 - [x] Added clear title that can be used to generate release notes

### CODE REVIEW

File: docs/source/news.rst [hunk 1]
```diff
@@ -21,6 +21,8 @@ Details about the build system unification can be found in :ref:`RFC7`.
 Updates
 -------
 
+* Docs: add new page Usage/Ellipsoids to explain the usage of ellipsoidal parameters
```
Comment (kbevers) [inline docs/source/news.rst]: We generally do not include doc updates in the NEWS section (unless they are quite significant) as they should be online at about the same time as they are merged.

**Comment by kbevers** [comment]
Thanks for taking this up again. In terms of the commit history this PR is a bit messy, so I'll just squash merge it when the time comes. That will make the two of you co-authors of that single commit, which I guess reflect the state of things quite well.

File: docs/images/general_ellipsoid.svg [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+       xml:space="preserve"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:4.93889px;line-height:1.25;font-family:'Linux Libertine Display O';-inkscape-font-specification:'Linux Libertine Display O, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;fill:#000000;fill-opacity:1;stroke:none;stroke-width:0.112651"
+       x="181.45177"
+       y="124.99306"
+       id="text885"><tspan
+         sodipodi:role="line"
+         x="181.45177"
+         y="124.99306"
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:4.93889px;font-family:'Linux Libertine Display O';-inkscape-font-specification:'Linux Libertine Display O, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-east-asian:normal;stroke-width:0.112651"
+         id="tspan887">Semi-major axis (a)</tspan></text>
```
Comment (mwtoews) [inline docs/images/general_ellipsoid.svg:127]: Maybe show a different line for half of this line. The major axis is the full line, while the _semi_-major is half. Same comment applies to the semi-minor axis in this figure.

**Review by mwtoews — COMMENTED** [review]
Hope you don't mind a commit here to update a few other things, including to indent the option and use math formatting. The comment in the code for "R_V" needed to be corrected, as it was an old typo.

**Comment by direvus** [comment]
Thanks for the additional markup improvements!

Good pickup on the "semi" axis labels too, I feel a bit silly for not noticing that.  Guess I was too focussed on the diagram design and not thinking critically about its contents.

I think splitting those axes in half and tuning down the opacity on the unlabelled half will do the trick.  I'll make that change on the branch.

File: docs/images/general_ellipsoid.svg (hunk 2 above)
Comment (direvus) [reply docs/images/general_ellipsoid.svg:127]: I have updated the diagram, thanks for your feedback.

File: docs/source/usage/ellipsoids.rst [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+    proj=laton R=7000000
+
+Using the GRS80 ellipsoid:
+
+::
+
+    proj=laton ellps=GRS80
+
+Expressing ellipsoid by semi-major axis and reverse flattening (1/f):
```
Comment (mwtoews) [inline docs/source/usage/ellipsoids.rst]:
```suggestion
Expressing ellipsoid by semi-major axis and reverse flattening (:math:`1/f`):
```

**Review by mwtoews — APPROVED** [review]
Just one missed formula, and undo the edit to `docs/source/news.rst` (suggested by @kbevers) and this will be nice to merge. Thanks for getting this one finished!

**Comment by direvus** [comment]
Thanks @mwtoews, I think I've got those final changes covered now.

**Pull request merged**